### ISSUE TITLE
Build from repository fails (for me) at autoreconf on OS X

### ISSUE THREAD

**Issue by jyrgenn** [simoncozens/sile#81]
OS X 10.10.3, automake 2.69 installed with Homebrew, sile as of ee1edbdcb53cd51e7c605caa9ef718f781fd293f (2015-05-12Z05:00).

`autoreconf` fails with lots of error messages when I run the following:
```
git clone https://github.com/simoncozens/sile.git
cd sile
autoreconf --version
sh -x ./bootstrap.sh
```
(See the output for details below.) The same command sequence works fine for me on Debian Jessie. Unfortunately I am devoid of any Automake knowledge. All I can imagine is a shell or m4 issue. The m4 is `/usr/bin/m4`, and Homebrew does not seem to offer m4.

```
: wrixum:0:.../ni/sw; git clone https://github.com/simoncozens/sile.git
Cloning into 'sile'...
remote: Counting objects: 4322, done.
remote: Compressing objects: 100% (135/135), done.
remote: Total 4322 (delta 78), reused 0 (delta 0), pack-reused 4187
Receiving objects: 100% (4322/4322), 21.38 MiB | 1.45 MiB/s, done.
Resolving deltas: 100% (2737/2737), done.
Checking connectivity... done.
: wrixum:0:.../ni/sw; cd sile
: wrixum:0:.../sw/sile; autoreconf --version
autoreconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.
: wrixum:0:.../sw/sile; sh -x ./bootstrap.sh 
+ git submodule update --init --recursive --remote
Submodule 'libtexpdf' (https://github.com/simoncozens/libtexpdf.git) registered for path 'libtexpdf'
Cloning into 'libtexpdf'...
remote: Counting objects: 2653, done.
remote: Total 2653 (delta 0), reused 0 (delta 0), pack-reused 2653
Receiving objects: 100% (2653/2653), 2.44 MiB | 421.00 KiB/s, done.
Resolving deltas: 100% (2097/2097), done.
Checking connectivity... done.
Submodule path 'libtexpdf': checked out '933e0a6129360ddcb092b7777c26921d1d23c7ff'
+ autoreconf --install
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
Makefile.am:3: error: Libtool library used but 'LIBTOOL' is undefined
Makefile.am:3:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
Makefile.am:3:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
Makefile.am:3:   If 'LT_INIT' is in 'configure.ac', make sure
Makefile.am:3:   its definition is in aclocal's search path.
autoreconf: automake failed with exit status: 1
+ cd libtexpdf
+ autoreconf -I m4
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2739: _AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2757: AC_RUN_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:643: AS_IF is expanded from...
../../lib/autoconf/general.m4:2046: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2059: AC_CACHE_CHECK is expanded from...
../../lib/autoconf/functions.m4:479: AC_FUNC_CLOSEDIR_VOID is expanded from...
m4/kpse-common.m4:264: KPSE_COMMON is expanded from...
configure.ac:16: the top level
Makefile.am:3: error: Libtool library used but 'LIBTOOL' is undefined
Makefile.am:3:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
Makefile.am:3:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
Makefile.am:3:   If 'LT_INIT' is in 'configure.ac', make sure
Makefile.am:3:   its definition is in aclocal's search path.
autoreconf: automake failed with exit status: 1
+ sed 's/rm -f core/rm -f/' configure
sed: configure: No such file or directory
+ mv config.cache configure
+ chmod +x configure
: wrixum:0:.../sw/sile; l
total 168
-rw-r--r--    1 ni  staff   1079 May 12 07:01 LICENSE
-rw-r--r--    1 ni  staff    184 May 12 07:01 Makefile.am
-rw-r--r--    1 ni  staff   1983 May 12 07:01 NEWS.md
-rw-r--r--    1 ni  staff   2942 May 12 07:01 README.md
-rw-r--r--    1 ni  staff    303 May 12 07:01 ROADMAP
-rw-r--r--    1 ni  staff  49941 May 12 07:01 aclocal.m4
drwxr-xr-x    7 ni  staff    238 May 12 07:01 autom4te.cache
-rwxr-xr-x    1 ni  staff    209 May 12 07:01 bootstrap.sh
drwxr-xr-x    7 ni  staff    238 May 12 07:01 classes
-rwxr-xr-x    1 ni  staff      0 May 12 07:02 configure
-rw-r--r--    1 ni  staff   3378 May 12 07:01 configure.ac
drwxr-xr-x   30 ni  staff   1020 May 12 07:01 core
drwxr-xr-x   10 ni  staff    340 May 12 07:01 documentation
drwxr-xr-x   40 ni  staff   1360 May 12 07:01 examples
drwxr-xr-x   59 ni  staff   2006 May 12 07:01 languages
drwxr-xr-x  138 ni  staff   4692 May 12 07:02 libtexpdf
drwxr-xr-x   13 ni  staff    442 May 12 07:01 lua-libraries
drwxr-xr-x    4 ni  staff    136 May 12 07:01 m4
drwxr-xr-x   35 ni  staff   1190 May 12 07:01 packages
-rwxr-xr-x    1 ni  staff   1206 May 12 07:01 sile.in
drwxr-xr-x    7 ni  staff    238 May 12 07:01 src
drwxr-xr-x   38 ni  staff   1292 May 12 07:01 tests
: wrixum:0:.../sw/sile; 
```

**Comment by simoncozens** [comment]
You have basically the same setup as I do; but it looks like autoconf is picking up macros from strange place, and not picking up macros. Do you have autoconf 2.69 from homebrew as well as automake?

**Comment by jyrgenn** [comment]
Yes, it is autoconf 2.69 and automake 1.15 (see below). Both versions seem to be current as trying `brew upgrade` on them shows. Both also seem to be the only installations AFAICS; I have removed previous Fink and MacPorts installations a long while ago.

If this is just due to a weirdness of my own installation, I'd be totally content with a WONTFIX WORKSFORME. I can build (actually, have built) and use the current SILE on the Debian box. That box is remote, but ssh/scp/rsync access is fast. And then there will be the 0.9.2 release. :–)

```
: wrixum:0:.../sw/sile; brew list autoconf
/usr/local/Cellar/autoconf/2.69/bin/autoconf
/usr/local/Cellar/autoconf/2.69/bin/autoheader
/usr/local/Cellar/autoconf/2.69/bin/autom4te
/usr/local/Cellar/autoconf/2.69/bin/autoreconf
/usr/local/Cellar/autoconf/2.69/bin/autoscan
/usr/local/Cellar/autoconf/2.69/bin/autoupdate
/usr/local/Cellar/autoconf/2.69/bin/ifnames
/usr/local/Cellar/autoconf/2.69/share/autoconf/ (42 files)
/usr/local/Cellar/autoconf/2.69/share/emacs/ (4 files)
/usr/local/Cellar/autoconf/2.69/share/info/autoconf.info
/usr/local/Cellar/autoconf/2.69/share/man/ (9 files)
: wrixum:0:.../sw/sile; brew list automake
/usr/local/Cellar/automake/1.15/bin/aclocal
/usr/local/Cellar/automake/1.15/bin/aclocal-1.15
/usr/local/Cellar/automake/1.15/bin/automake
/usr/local/Cellar/automake/1.15/bin/automake-1.15
/usr/local/Cellar/automake/1.15/share/aclocal/ (2 files)
/usr/local/Cellar/automake/1.15/share/aclocal-1.15/ (33 files)
/usr/local/Cellar/automake/1.15/share/automake-1.15/ (76 files)
/usr/local/Cellar/automake/1.15/share/doc/automake/amhello-1.0.tar.gz
/usr/local/Cellar/automake/1.15/share/info/ (4 files)
/usr/local/Cellar/automake/1.15/share/man/ (4 files)
```

**Comment by simoncozens** [comment]
Well, I know it works for me but that's no solution - it might be a weirdness of *my* own installation. :-) I'm going to set up a clean virtualbox OS X install and see if I can get it working on that.

**Comment by simoncozens** [comment]
OK, spinning up a clean VM and installing as per the instructions, I find:

* I also need to install pkg-config
* I need to set PKG_CONFIG_PATH appropriately: `PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib`

and then it configures properly. Try those two things, I'll update the documentation, and if it doesn't work for you - sorry, it probably is your system.

**Comment by jyrgenn** [comment]
It doesn't indeed. Apparently I had `pkg-config` already installed —
```
: wrixum:0:.../Documents/dokomente; brew install pkg-config
Warning: pkg-config-0.28 already installed
```
— and I have changed `bootstrap.sh` to set `PKG_CONFIG_PATH`:
```
: wrixum:127:.../sw/sile; cat bootstrap.sh 
#!/bin/sh
PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib
export PKG_CONFIG_PATH
git submodule update --init --recursive --remote
autoreconf --install
(cd libtexpdf; autoreconf -I m4)
sed 's/rm -f core/rm -f/' configure > config.cache
mv config.cache configure
chmod +x configure
```
Still I get the same symptoms.
```
+ PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib
+ export PKG_CONFIG_PATH
+ git submodule update --init --recursive --remote
Submodule 'libtexpdf' (https://github.com/simoncozens/libtexpdf.git) registered for path 'libtexpdf'
Cloning into 'libtexpdf'...
remote: Counting objects: 2653, done.
remote: Total 2653 (delta 0), reused 0 (delta 0), pack-reused 2653
Receiving objects: 100% (2653/2653), 2.44 MiB | 595.00 KiB/s, done.
Resolving deltas: 100% (2097/2097), done.
Checking connectivity... done.
Submodule path 'libtexpdf': checked out '933e0a6129360ddcb092b7777c26921d1d23c7ff'
+ autoreconf --install
configure.ac:16: warning: AC_REQUIRE: `AC_HEADER_STDC' was expanded before it was required
configure.ac:16: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/headers.m4:341: _AC_INCLUDES_DEFAULT_REQUIREMENTS is expanded from...
../../lib/autoconf/headers.m4:357: AC_INCLUDES_DEFAULT is expanded from...
../../lib/autoconf/lang.m4:225: AC_LANG_SOURCE is expanded from...
../../lib/autoconf/lang.m4:242: AC_LANG_PROGRAM is expanded from...
../../lib/autoconf/lang.m4:197: AC_LANG_CONFTEST is expanded from...
```
and so on, as above.

If you say that is my system, that may well be. I will try to do a fresh installation with VirtualBox myself.

**Comment by jyrgenn** [comment]
I have now done a fresh Yosemite installation in a VirtualBox VM and get the same errors. These are the full command histories of the two shells I used to set this up; except for the `bootstrap.sh` and some irrelevant things there were no errors in the output.

Am I doing something wrong? 

     1  wget

(has no wget in the system)

     2  curl

(but has curl)

     3  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     4  brew install luarocks

(Didn't work)

     5  brew install lua
     6  git
     7  type git
     8  brew install harfbuzz

(Error due to missing XQuartz; installed XQuartz)

     9  ifconfig -a
    10  brew install harfbuzz

(Oops, XQuartz installer was started, but not done.)

    11  brew install harfbuzz
    12  brew install lua
    13  brew install luarocks

(Still doesn't work)

    14  brew update

(Switched to other shell; ssh'ed into the VM works better than its slow graphics console)

     1  brew install luarocks

(Still doesn't work although it *should*, says the luarocks wiki)

     2  brew install wget
     3  wget http://keplerproject.github.io/luarocks/releases/luarocks-2.2.2.tar.gz
     4  l
     5  ls -al
     6  tar ztvf luarocks-2.2.2.tar.gz 
     7  tar zxvf luarocks-2.2.2.tar.gz 
     8  cd luarocks-2.2.2
     9  l
    10  ;;

(oops)

    11  ll
    12  ls -al

(my ls aliases are not in this shell, darn)

    13  ./configure
    14  make build
    15  make install
    16  luarocks install lpeg
    17  luarocks install luaexpat
    18  luarocks install lgi
    19  brew install pkg-config
    20  git
    21  git clone https://github.com/simoncozens/sile
    22  l
    23  ls 
    24  rm -rf sile

(that was the wrong place; actually mv would have sufficed)

    25  cd
    26  brew install autoconf
    27  brew install automake
    28  git clone https://github.com/simoncozens/sile
    29  ls
    30  cd sile
    31  ls
    32  sh -x ./bootstrap
    33  sh -x ./bootstrap.sh 

(once you do it right..., but still the errors)

    34  ls -al
    35  cd ..
    36  rm -rf sile
    37  git clone https://github.com/simoncozens/sile
    38  cd sile
    39  vi bootstrap.sh 

(had forgotten the PKG_CONFIG_PATH)

    40  sh -x ./bootstrap.sh 
 
At this point (with the  it should work, but I still get the same errors and an empty configure script. Ist it my bad aura?

Anyway, if you don't see what is going wrong here, I don't really need to build from the repository, so I'll gladly wait for the next release and in between build a fresh one on Debian, if necessary. This yak has been getting bigger and bigger, and my motivation to pull this through has shrunk.

**Comment by simoncozens** [comment]
Closing, then. I've updated the documentation, and released versions don't need autotools, 0.9.3 will be out soon.

**Comment by jyrgenn** [comment]
That is probably the right thing to do. While I'd love to sit down with you and check what we're doing differently setting up a clean VM for a build, I am afraid that is a bit on the impractical side, and not only because I am on vacation on Mallorca right now. ;–)

Regards, Juergen.
